dts/bindings/sensor/avia,hx711.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2020 George Gkinis
+# Copyright (c) 2021 Jan Gnip
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Avia HX711 sensor
+
+compatible: "avia,hx711"
+
+include: avia,hx7xx-common.yaml
+
+properties:
+  gain:
+    type: int
+    description: |
+      Gain of the sensor.
+      Default set to 128x as this is the sensors default setting after power-up.
+    default: 128
+    enum: [128, 32, 64]
+
+  rate-hz:
+    type: int
+    description: |
+      Sensor sample rate.
+      User should provide initial value depending on HW implementation.
+      If CONFIG_PM_DEVICE_RUNTIME = y rate will not be higher
+      than 2~3Hz due to the long wake-up time needed for HX711.
+    default: 10
+    enum: [10, 80]
+
+  rate-gpios:
```

Comment:
It would be nice if this was optional. Some implementations might not use a GPIO for this pin and instead pull-up or pull-down.